mappings/net/minecraft/block/Block.mapping

```diff
@@ -25,6 +25,12 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 	METHOD method_23350 getJumpVelocityMultiplier ()F
 	METHOD method_27839 is (Lnet/minecraft/class_2248;)Z
 		ARG 1 block
+	METHOD method_30094 replaceBlock (Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;I)V
```

Comment:
`replaced`?

Reply:
If this is called when this block is replaced, I'd suggest
```suggestion
	METHOD method_30094 replaced (Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;I)V
```

Reply:
This is the exact same name as the other method that also passes the maximum depth. Unless we want to change that one also (not an issue tbf)

Reply:
Hmm, how often is this method overridden by mods? Need to weigh the benefit and the cost.

Reply:
This is a static method